sdk-quarkus/kar-rest-client/mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
I know quarkus generates this in its sample project, but I think we don't actually need it (we pre-req maven being installed for other parts of kar)